.github/workflows/pr.yml

```diff
@@ -3,125 +3,108 @@ on: pull_request
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v4
-
-            - name: Test suite
-              run: |
-                  docker network create frontend
-                  docker compose run --rm phpfpm composer install
-                  docker compose run -e XDEBUG_MODE=coverage --rm phpfpm vendor/bin/phpunit --coverage-clover=coverage/unit.xml
-
-            - name: Upload coverage to Codecov test
-              uses: codecov/codecov-action@v2
-              with:
-                  files: ./coverage/unit.xml
-                  flags: unittests, ${{ matrix.php }}
+  test-composer-install:
+    runs-on: ubuntu-latest
+    name: Validate composer
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          docker network create frontend
+          docker compose run --rm phpfpm composer validate composer.json --strict
+          docker compose run --rm phpfpm composer install
+          docker compose run --rm phpfpm composer normalize --dry-run
+          docker compose run --rm phpfpm composer audit
+
+  validate-doctrine-schema:
+    runs-on: ubuntu-latest
+    name: Validate Doctrine Schema
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          docker network create frontend
+          docker compose run --rm phpfpm composer install
+          docker compose run --rm phpfpm bin/console doctrine:migrations:migrate --no-interaction
+          docker compose run --rm phpfpm bin/console doctrine:schema:validate
+
+  code-analysis:
+    name: PHP - Code analysis (phpstan)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          docker network create frontend
+          docker compose run --rm phpfpm composer install
+          docker compose run --rm phpfpm composer code-analysis
+
+  coding-standards-phpcs:
+    name: coding-standards-check/phpcs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          docker network create frontend
+          docker compose run --rm phpfpm composer install
+          docker compose run --rm phpfpm composer coding-standards-check
+
+  assets-coding-standards:
+    name: Assets coding standards
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Coding standards
+        run: |
+          docker network create frontend
+          docker compose run --rm node npm install
+          docker compose run --rm node npm run actions/check-coding-standards-assets
+
+  markdown-coding-standards:
+    name: Markdown coding standards
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Coding standards
+        run: |
+          docker network create frontend
+          docker compose run --rm node npm install
+          docker compose run --rm node npm run coding-standards-check/markdownlint
+
+  twig-coding-standards:
```

Comment:
Duplicate of .github/workflows/twig.yaml

Make sure none of the checks in pr.yaml are handled in one of the other workload files.